x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx

```diff
@@ -192,10 +196,35 @@ const EventDetailsComponent: React.FC<Props> = ({
                 {enrichmentsLoading ? <EuiLoadingSpinner /> : `(${enrichmentCount})`}
               </span>
             ),
-            content: <ThreatDetailsView enrichments={allEnrichments} />,
+            content: (
+              <>
+                <EuiSpacer size="l" />
+                {existingEnrichments.length < 1 && <NoIndicatorRuleEnrichmentsPanel />}
+                {!isEmpty(eventFields) && !investigationEnrichments?.length && (
```

Comment:
The `isEmpty` guard here prevents us from displaying this panel if the alert has no relevant fields to search against. We could alternately handle this case with a different callout.